<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>win_main</class>
 <widget class="QMainWindow" name="win_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1273</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1273</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="action_ffnen"/>
    <addaction name="separator"/>
    <addaction name="actionBeenden"/>
   </widget>
   <widget class="QMenu" name="menuFenster">
    <property name="title">
     <string>Fenster</string>
    </property>
    <addaction name="actionVollbild"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuFenster"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBeenden">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
  <action name="action_ffnen">
   <property name="text">
    <string>Ã–ffnen</string>
   </property>
  </action>
  <action name="actionVollbild">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Vollbild</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
